integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/ssh/action/SshJClientActions.java

```diff
@@ -310,17 +315,174 @@ public DistroXTestDto checkMeteringStatus(DistroXTestDto testDto, List<InstanceG
                     if (StringUtils.containsIgnoreCase(event, "databusReachable")) {
                         Log.log(LOGGER, format(" Found 'databusReachable' status is not OK at '%s' instance. However this is acceptable!",
                                 meteringStatusReport.getKey()));
-                        LOGGER.warn("Found 'databusReachable' status is not OK at '{}' instance. However this is acceptable!", meteringStatusReport.getKey());
                     } else {
-                        Log.log(LOGGER, format(" Found '%s' not OK at '%s' instance. ", event, meteringStatusReport.getKey()));
-                        LOGGER.error("There is 'Not OK' Metering Heartbeat status {} is present on '{}' instance!", event, meteringStatusReport.getKey());
+                        Log.error(LOGGER, format(" There is 'Not OK' Metering Heartbeat status %s is present on '%s' instance! ", event,
+                                meteringStatusReport.getKey()));
                         throw new TestFailException(format("There is 'Not OK' Metering Heartbeat status %s is present on '%s' instance!", event,
                                 meteringStatusReport.getKey()));
                     }
                 });
             }
         }
+        return testDto;
+    }
+
+    /**
+     * Creating new pre-warmed images with Common Monitoring is still in progress. So in the meantime we can install latest version on the VMs manually.
+     *
+     * @param instanceGroups An instance group is a collection of virtual machine (VM) instances that you can manage as a single entity.
+     * @param hostGroupNames A host group logically grouped hosts regardless of any features that they might or might not have in common.
+     *                          For example: hosts do not have to have the same architecture, configuration, or storage.
+     */
+    private void installMonitoringSnapshot(List<String> instanceIps) {
+        String installTelemetrySnapshot =
+                "sudo /opt/salt/scripts/cdp-telemetry-deployer.sh upgrade -c cdp-telemetry -v snapshot > /dev/null 2>&1";
+        String removeOldTelemetry = "sudo yum remove -y cdp-telemetry > /dev/null 2>&1";
+        String addTelemetrySnapshotPackage =
+                "sudo rpm -i https://cloudera-service-delivery-cache.s3.amazonaws.com/telemetry/cdp-telemetry/cdp_telemetry-0.1.0-SNAPSHOT.x86_64.rpm " +
+                        "> /dev/null 2>&1";
+
+        instanceIps.stream()
+                .collect(Collectors.toMap(ip -> ip, ip -> executeSshCommand(ip, StringUtils.join(List.of(installTelemetrySnapshot, removeOldTelemetry,
```

Comment:
it should work now without the removeOldTelemetry and addTelemetrySnapshotPackage command